index.html

```diff
@@ -14,9 +14,43 @@
   </head>
   <body>
     <header>
-      
+      <nav>
+        <a id="logo" href="/">My Logo</a>
+        <span id="menu-btn">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke-width="1.5"
+            stroke="currentColor"
+            class="w-6 h-6"
+          >
+            <path
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
+            />
+          </svg>
+        </span>
+      </nav>
     </header>
-    <main></main>
+    <main>
+      <section id="headline">
+        <h1 id="primary-text">I'm Esteban. Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+        <p>LET'S CONNECT</p>
+        <img
+          class="social icon"
+          src="assets/headline/social media icons.svg"
+          alt="twitter link"
```

Comment:
- [ ] kindly consider adding these icons in `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).instead of adding the as a single image

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. remember to align them using flexbox too